product-scenarios/5-route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/5.1.1-Route-messages-based-on-the-given-xpath/5.1.1.1-Using-switch-mediator-on-given-xpath/src/test/resources/source_files/5.1.1.1.1/5.1.1.1.1/request/5.1.1.1.1.xml

```diff
@@ -0,0 +1,3 @@
+<m:GetStockPrice xmlns:m="http://www.example.org/stock">
```

Comment:
Seems like these files are nt being used. Could you double check?

Reply:
removed unnecessary files.